content/cockpit-129.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 128 and 129
+author: Dominik Perpeet
+date: 2016-01-19 09:00
+tags: cockpit, linux
+slug: cockpit-129
+category: release
+summary: Cockpit supports Rebases and Remotes in OSTree.
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release
+regularly. Here are the release notes from version 128 and 129.
+
+### Manage remotes and rebase in OSTree
+
+When working with [OSTrees](http://www.projectatomic.io/docs/os-updates/) there will
+often be multiple branches to choose from. Thanks to Peter's work, Cockpit can now switch
+between branches and view and activate OSTrees from those branches. Also, Cockpit supports
+managing multiple remotes and viewing their branches.
+Take a look at the video below for a demo.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/dF8M3xq9FcI" frameborder="0" allowfullscreen></iframe>
+
+### cockpit-dashboard split out
+
+The new ```cockpit-dashboard``` subpackage contains the dashboard itself and
+the ```cockpit-ssh``` process. Eventually this paves the way for more flexibility
+regarding authentication processes, but for now ```cockpit-ws``` unconditionally
+depends on ```cockpit-dashboard```, and also requires the identical versions.
+
```

Comment:
"Nothing changes for those who install the ```cockpit``` package. But this allows more flexibility when using Cockpit for specific use cases."